src/pages/api/poap/get-withdrawable-amount/[guildId]/[poapId].ts

```diff
@@ -18,11 +18,15 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
   if (!guildId || !poapId)
     return res.status(400).json({ error: "Missing parameters" })
 
-  const guild: Guild = await fetcher(`/guild/${guildId}`).catch((_) => ({}))
+  const guild: Guild = await fetcher(`/v2/guilds/${guildId}`).catch((_) => ({}))
 
   if (!guild.id) return res.status(404).json({ error: "Guild not found." })
 
-  const poap = guild.poaps?.find((p) => p.id.toString() === poapId)
+  const guildPoaps: Guild["poaps"] = await fetcher(
+    `/v2/guilds/${guildId}/poaps`
+  ).catch((_) => ({}))
```

Comment:
I think this should be an empty array, actually?

Reply:
Nice catch!
Changed it to an empty array in https://github.com/agoraxyz/guild.xyz/pull/894/commits/a45c7c35cadc3daf104361bd310724ee82548acb!